src/src/components/ingest/publications_edit.jsx

```diff
@@ -1877,8 +1877,8 @@ class PublicationEdit extends Component {
 
   isAssayCheckSet(assay) {
     try {
-      if (this.props.editingPublication.data_types) {
-        return this.props.editingPublication.data_types.includes(assay);
+      if (this.props.editingPublication.dataset_type) {
+        return this.props.editingPublication.dataset_type.includes(assay);
```

Comment:
Will this still work when `dataset_type` is a string?

Reply:
the "isAssayCheckSet" function this is within is never used, holdover from dataset form